src/components/editor-page/hooks/useUpdateHistoryEntry.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import equal from 'fast-deep-equal'
+import { useCallback, useEffect, useRef } from 'react'
+import { useSelector } from 'react-redux'
+import { ApplicationState, store } from '../../../redux'
+import { useParams } from 'react-router-dom'
+import { EditorPagePathParams } from '../editor-page'
+import { HistoryEntry, HistoryEntryOrigin } from '../../../redux/history/types'
+import { updateLocalHistoryEntry } from '../../../redux/history/methods'
+
+export const useUpdateHistoryEntry = (loading: boolean, error: boolean): void => {
```

Comment:
Instead of providing both variables I think it's a better idea to provide a more meaningful variable like `updateEnabled`. This hook doesn't care why it shouldn't update. It's only important THAT it shouldn't update.

Reply:
Changed.